.github/workflows/build.yml

```diff
@@ -0,0 +1,34 @@
+name: Multi-OS Build
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request: 
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Cache Gradle packages
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+      - name: Build with Gradle
+        run: ./gradlew clean build
```

Comment:
Should it be `./gradlew test`?
Will this command work on Windows?

Reply:
We could call ./gradlew test as well. Actually the "build" gradle task calls the "test" gradle task, but it also produces the jar. 
So, since we do nothing we the jar, we could run only the test task if you'd prefer it.

And yes, this command works on windows as well. Here's the link to my fork of this repository, on the "actions" page, latest build. As you can see 3 builds (+tests) were triggered by the build.yaml. One for each OS, and the three of them are successful
https://github.com/GuillaumeTaffin/matlab-IntelliJ-plugin/actions/runs/439337784

Reply:
Changed for only caling the test task
https://github.com/GuillaumeTaffin/matlab-IntelliJ-plugin/actions/runs/442646867